trio/_tests/test_deprecate.py

```diff
@@ -259,7 +259,7 @@ def test_module_with_deprecations(recwarn_always: pytest.WarningsRecorder) -> No
     assert "instead-string instead" in got.message.args[0]
 
     with pytest.raises(AttributeError):
-        module_with_deprecations.asdf  # type: ignore[attr-defined]  # noqa
+        module_with_deprecations.asdf  # type: ignore[attr-defined]  # noqa: B018
```

Comment:
`# "useless expression"`

(I really dislike that flake8 doesn't have name-based noqas)

Reply:
Added to comment in a5afed7